.github/workflows/canary-integration-test.yml

```diff
@@ -49,6 +49,11 @@ jobs:
         docker images
         docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
 
+    - name: yaml linter
+      run: |
+        sudo apt-get install yamllint -y
+        yamllint -d "{extends: relaxed, rules: {line-length: disable}}" . --no-warnings
```

Comment:
if we don't want to run one more actions then we can also do it in this way... @travisn @BlaineEXE let me know which is better

Reply:
A separate action sounds good to me. This doesn't have any dependency on k8s or the integration tests.